seismicpro/src/refractor_velocity/refractor_velocity.py

```diff
@@ -492,6 +511,67 @@ def _ms_to_kms(self, params, as_array=True):
             return values
         return dict(zip(self._valid_keys, values))
 
+    def dump(self, path, col_size=10, encoding="UTF-8"):
+        """Dump RefractorVelocity instance to disk.
+
+        File contains coords and params attributes with follow structure:
+        [coords.names] [params.keys]
+        [coords.coords] [params.values]
+
+        Parameters
+        ----------
+        path : str,
+            path to the file with parameters.
```

Comment:
I've stolen this from gather.dump, seems better for me.
Also, make sure that the sentences start with capital letters
```suggestion
            The directory to dump the RefractorVelocity instance in.
```

Reply:
It's full path to a file